omnigibson/simulator.py

```diff
@@ -363,8 +363,8 @@ def __init__(
                     position=np.array(m.DEFAULT_VIEWER_CAMERA_POS),
                     orientation=np.array(m.DEFAULT_VIEWER_CAMERA_QUAT),
                 )
-                self.viewer_width = viewer_width
-                self.viewer_height = viewer_height
+                # self.viewer_width = viewer_width
+                # self.viewer_height = viewer_height
```

Comment:
Figure out why uncommenting this breaks things.

Reply:
This is super confusing but I just figured this out: essentially the problem here is the new replicator api does not support having two render products at the same time. Our code used to hold the old render product and the new render product at the same time to detach and attach all our annotators. Now, we will have to detach all the annotators from the old render product, destroy the old one, create a new one, and attach the annotators.